### DISCUSSION TITLE
Stylelint rule error because keyframes are not available in atomic generated file

### DISCUSSION THREAD

**Discussion by Aakashsharma27** [acss-io/atomizer#560] [Q&A]
Since there is no way to create keyframes in atomizer but we can use 'animation-name' property in it. 

<img width="617" alt="image" src="https://user-images.githubusercontent.com/120000160/206399415-3329c212-8afb-46f9-a2de-dd7fd4d86a95.png">

I am running stylelint on generated atomic css

Now that I have integrated "[no-unknown-animation](https://stylelint.io/user-guide/rules/no-unknown-animations/)" stylelint rule in my code and it expects keyframe in same css file and as we know keyframe can't be created in generated acss file.

I am getting that error to define keyframes in atomic css.

Is there a way to solve that problem?

<img width="670" alt="Screenshot 2022-12-08 at 1 36 22 PM" src="https://user-images.githubusercontent.com/120000160/206393115-8a987f24-fe26-4832-8356-8b44f758abe0.png">

Thanks

**Comment by redonkulus** [comment]
Unfortunately no, since Atomizer does not have a way to define keyframes via atomizer classes.
